client/routes.js

```diff
@@ -14,20 +13,25 @@ class Routes extends Component {
   }
 
   render() {
-    const {isLoggedIn} = this.props
+    const { isLoggedIn } = this.props
+    //console.log(this.props);
 
     return (
       <div>
         {isLoggedIn ? (
           <Switch>
-            <Route path="/home" component={Home} />
-            <Route path='/products' component={AllProducts}/>
+            <Route path='/home' component={Home} />
+            <Route exact path='/admin' component={AdminTools} />
+            <Route exact path='/products' component={AllProducts} />
+            <Route path='/products/create' component={productForm} />
+            <Route path='/products/update/:id' component={productForm} />
           </Switch>
         ) : (
           <Switch>
-            <Route path="/login" component={Login} />
-            <Route path="/signup" component={Signup} />
-            <Redirect to="/login" />
+            <Route path='/login' component={Login} />
+            <Route path='/signup' component={Signup} />
+            <Route path='/home' component={Home} />
```

Comment:
Add comment here that we will be displaying home and products for non logged-in users, and that we need to add the functionality in order to tell a user to log in if they try to create an order